crypto/provider_core.c

```diff
@@ -278,6 +296,28 @@ void ossl_provider_free(OSSL_PROVIDER *prov)
     }
 }
 
+/* Setters */
+int ossl_provider_set_module_path(OSSL_PROVIDER *prov, const char *module_path)
+{
+    prov->path = OPENSSL_strdup(module_path);
+    return 1;
+}
+
+int ossl_provider_add_parameter(OSSL_PROVIDER *prov,
+                                const char *name, const char *value)
+{
+    INFOPAIR *pair = OPENSSL_malloc(sizeof(*pair));
```

Comment:
Missing NULL check after this allocation.

Reply:
Fixed